index.html

```diff
@@ -1,9 +1,76 @@
 <html>
-    <head>
-        <title>Hello</title>        
-        <script src="js/main.js" type="text/javascript"></script>
-    </head>
-    <body>
-        <div>Hello</div>
-    </body>
+  <head>
+    <title>Semantic elements</title>        
+    <link href="style/main.css" rel="stylesheet" type="text/css"/>
+    <script src="js/main.js" type="text/javascript"></script>
+  </head>
+  <body>
+    <header id="header">
+      <div id="logo"><a href="#">&reg;</a></div>
+      <nav>
+        <ul>
+          <li><a href="#">Main</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Other</a></li>
+          <li><a href="#">Links</a></li>
+        </ul>
+      </nav>
+      <div class="clear"></div>
+    </header>
+    <main id="main">
+      <h1>Main</h1>
+      <section>
+        <header>Header</header>
+        <main>
+          <figure>            
+            <img src="style/img/img.png" width="128" height="128" alt="img"/>            
+            <figcaption>
+              Image
+            </figcaption>
+          </figure>
+          <p><mark>Lorem Ipsum</mark> is simply dummy text of the printing and typesetting
+              industry. Lorem Ipsum has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a galley of
+              type and scrambled it to make a type specimen book. It has
+              survived not only five centuries, but also the leap into
+              electronic typesetting, remaining essentially unchanged. It was
+              popularised in the 1960s with the release of Letraset sheets
+              containing Lorem Ipsum passages, and more recently with desktop
+              publishing software like Aldus PageMaker including versions of
+              Lorem Ipsum.
+            </p>
+        </main>        
+        <footer>Footer</footer>
+        <aside></aside>
```

Comment:
imo aside should be outside section, you will have a problem with css with this structure